e2e-tests/stream-mode-tests.sh

```diff
@@ -0,0 +1,245 @@
+    },
+    "identifier": [
+        {
+            "id": "05a29f94-c0ed-11e2-94be-8c13b969e334",
+            "extension": [
+                {
+                    "url": "http://fhir.openmrs.org/ext/patient/identifier#location",
+                    "valueReference": {
+                        "reference": "Location/8d6c993e-c2cc-11de-8d13-0010c6dffd0f",
+                        "type": "Location",
+                        "display": "Unknown Location"
+                    }
+                }
+            ],
+            "use": "official",
+            "type": {
+                "text": "OpenMRS ID"
+            },
+            "value": "10002T"
+        }
+    ],
+    "active": true,
+    "name": [
+        {
+            "id": "9d550771-11d6-41b6-89c1-893ecd78210b",
+            "family": "Clive",
+            "given": [
+                "Jita"
+            ]
+        }
+    ],
+    "gender": "male",
+    "birthDate": "1971-04-11",
+    "deceasedBoolean": false,
+    "address": [
+        {
+            "id": "f4ba90db-e412-4aa3-a519-2c33eac3e867",
+            "extension": [
+                {
+                    "url": "http://fhir.openmrs.org/ext/address",
+                    "extension": [
+                        {
+                            "url": "http://fhir.openmrs.org/ext/address#address1",
+                            "valueString": "Address17001"
+                        }
+                    ]
+                }
+            ],
+            "use": "home",
+            "city": "City7001",
+            "state": "State7001",
+            "postalCode": "47002",
+            "country": "Country7001"
+        }
+    ]
+}'
+
+print_message "CREATE AN ENCOUNTER IN OPENMRS TO BE STREAMED"
+
+curl -i -u admin:Admin123 -H "Content-type: application/json" -X POST http://localhost:8099/openmrs/ws/rest/v1/encounter -d '{
+"encounterDatetime":"2020-09-16T14:08:40.000+0530",
+"patient":"957d5571-dee2-4e5e-8ece-b65d2d965248",
+"location":"aff27d58-a15c-49a6-9beb-d30dcfc0c66e",
+"encounterType":"e22e39fd-7db2-45e7-80f1-60fa0d5a4378",
+"encounterProviders": [
+    {
+      "provider": "fb6adae0-191d-4d5f-9c7a-54af74270c67",
+      "encounterRole": "240b26f9-dd88-4172-823d-4a8bfeb7841f"
+    }
+  ]
+}'
+
+print_message "CREATE AN OBSERVATION IN OPENMRS TO BE STREAMED"
+
+curl -i -u admin:Admin123 -H "Content-type: application/json" -X POST http://localhost:8099/openmrs/ws/fhir2/R4/Observation -d '{
+
```

Comment:
i think you can place this resource somewhere in a json file and reference it by curl --data "@/path/to/filename"